src/app/shared/components/smart-search/smart-search.component.ts

```diff
@@ -14,6 +15,9 @@ export class SmartSearchComponent implements OnInit, AfterViewInit {
     @Input() searchText: FormControl;
     @Input() placeholderSearch: string;
     @Input() fieldName: string;
+    @Input() SearchType = "restrictions";
+
+    @Output() SearchTextEvent: EventEmitter<any> = new EventEmitter();
```

Comment:
camelCase please. Remove any and put there 
```suggestion
    @Output() searchText: EventEmitter<string> = new EventEmitter();
```